model_analyzer/config/input/config_defaults.py

```diff
@@ -51,12 +51,17 @@
 DEFAULT_RUN_CONFIG_MIN_MODEL_BATCH_SIZE = 1
 DEFAULT_RUN_CONFIG_MAX_MODEL_BATCH_SIZE = 128
 DEFAULT_RUN_CONFIG_MAX_BINARY_SEARCH_STEPS = 5
+DEFAULT_RUN_CONFIG_MIN_PROMPT_LENGTH = 1
+DEFAULT_RUN_CONFIG_MAX_PROMPT_LENGTH = 1000
+DEFAULT_RUN_CONFIG_MIN_TOKEN_COUNT = 1
+DEFAULT_RUN_CONFIG_MAX_TOKEN_COUNT = 256
```

Comment:
General question: What is MA trying to do by searching through different prompt sizes and token counts (aka. max_tokens)? Is it trying to find a good prompt size and token count?

Reply:
Yes, MA will sort the results by avg. first token-to-token latency in a table. But, the real benefit will be the graph that shows these values on the x-axis and avg. first token-to-token latency on the y-axis.